R/ui.R

```diff
@@ -47,6 +33,9 @@
 #' @param reference_values (Advanced Usage) Defines reference
 #'     dataset(s).  Cells that have batch variables values matching
 #'     reference_values will not be moved.
+#' @param .options Advanced parameters of HarmonyMatrix. This must be the 
```

Comment:
Add a small example on some valid form of the `.options` parameter so users do not have to look it up separately.

Reply:
I'm not sure about this. In general, we don't want users to touch parameters set by .options. And if they really want to do that as advanced users, they should really refer to the docs of harmony_options()